client/gutenberg/extensions/slideshow/edit.js

```diff
@@ -73,14 +81,22 @@ class SlideshowEdit extends Component {
 			onFileChange: images => {
 				const imagesNormalized = images.map( image => pickRelevantMediaFiles( image ) );
 				setAttributes( {
-					images: currentImages.concat( imagesNormalized ),
+					images: imagesNormalized.concat( currentImages ),
```

Comment:
What about

```suggestion
					images: [ ...imagesNormalized, ...currentImages ],
```

instead?

Reply:
Yes, much nicer! https://github.com/Automattic/wp-calypso/pull/30633/commits/371b7074e371e996d038a8e344d215fcbc5664cf